MyCryptoHQ/MyCrypto
# Fix getting stuck pending when node load fails.

### Description

Fixes an untracked issue where loading a node that fails causes the node pending state to be stuck, not allowing you to change nodes unless you refresh. I had forgotten to add handling for the failure in redux state.

### Steps to Test

1. Fudge one of the node URLs in `libs/nodes/config.ts` or go offline
2. Attempt to switch nodes
3. Confirm that it fails, but the node selector stops flashing
